shared/js/companies.js

```diff
@@ -106,8 +106,18 @@ var Companies = (() => {
             utils.syncToStorage({'lastStatsResetDate': lastStatsResetDate})
         },
 
+        sanitizeData: (storageData) => {
+            if (storageData.hasOwnProperty('twitter')) {
+              delete storageData.twitter
```

Comment:
Dropping lowercase twitter entirely, since it seems to usually appear on the same pages that have the Twitter tracker entry. 
This way global stats will be more reliable.